test_haystack/core/urls.py

```diff
@@ -1,28 +1,26 @@
 # encoding: utf-8
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import unicode_literals
 
-from django.conf.urls import include, patterns, url
+from django.conf.urls import include, url
 from django.contrib import admin
 
+from haystack import views
 from haystack.forms import FacetedSearchForm
 from haystack.query import SearchQuerySet
-from haystack.views import FacetedSearchView, SearchView
```

Comment:
I would prefer to revert this change because it allows static linters to verify that referenced variables exist without having to load views.py